packages/shared/src/variants/quantum.ts

```diff
@@ -32,18 +33,18 @@ class BadukHelper {
     return captures;
   }
 
-  clear(group: Coordinate[]) {
+  clear(group: CoordinateLike[]) {
     for (const pos of group) {
       this.badukGame.board.set(pos, Color.EMPTY);
     }
   }
 }
 
-export class QuantumGo extends AbstractGame<BadukConfig, QuantumGoState> {
+export class QuantumGo extends AbstractGame<GridBadukConfig, QuantumGoState> {
```

Comment:
Doesn't have to change in this PR, but I think QuantumGo could be adapted to graph boards.

Reply:
That's the plan eventually! I believe this will work as soon as I changed the vue-client components for rendering the board for Baduk. But that's for another PR.